edi_oca/security/ir_model_access.xml

```diff
@@ -72,17 +72,26 @@
         <field name="perm_write" eval="1" />
         <field name="perm_unlink" eval="1" />
     </record>
+    <record model="ir.model.access" id="access_edi_exchange_record_create_user">
+        <field name="name">access_edi_exchange_record user</field>
```

Comment:
what is this? Users can alread create records w/ the rule above.

Reply:
This rule is for the wizard, on 14.0 we need to add permissions for the wizards too :wink:

Reply:
ahh sorry, I was looking only at the diff of the mig commit and I missed that the wiz name is `edi.exchange.record.create`. 
Not a blocker, but IMO it should be renamed to `edi.exchange.record.create.wiz`. What do you think?

Reply:
sure!